tests/snapshots/index.js.md

```diff
@@ -123,7 +147,20 @@ Generated by [AVA](https://avajs.dev).
 > stdout
 
     `repositories not set, creating token for all repositories for given owner "smockle"␊
+    ␊
+    GET /users/smockle/installation␊
+    ␊
+    ␊
+    GET /users/smockle/installation␊
+    ␊
+    ␊
+    GET /users/smockle/installation␊
```

Comment:
I don't know why the request appears to be sent 3 times here, should just be twice.